aspnetcore/fundamentals/startup.md

```diff
@@ -59,6 +59,34 @@ For features that require substantial setup, there are `Add[Service]` extension
 
 [!code-csharp[](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
+::: moniker range=">= aspnetcore-2.1"
+
+<a name="setcompatibilityversion"></a>
+### SetCompatibilityVersion
+
+The `SetCompatibilityVersion` method allows an app to opt-in or opt-out of breaking changes in ASP.NET Core APIs. By opting in, you get the latest behavior, and the long term behavior of ASP.NET Core.
```

Comment:
> opt-out of breaking changes in ASP.NET Core APIs

Consider rewording this... We don't generally change how APIs behave because that will dramatically impact the code that people right. Generally these changes are in how the subsystems behave and how **your code** is called.